src/richchk/mpq/stormlib/stormlib_finder.py

```diff
@@ -18,6 +18,7 @@ class StormLibFinder:
     _MAC_STORM_INTEL = os.path.join(_SCRIPT_PATH, "dlls/macos/libStorm.dylib")
     _MAC_STORM_M1 = os.path.join(_SCRIPT_PATH, "dlls/macos/libstorm.9.22.0.dylib")
     _WINDOWS_STORM = os.path.join(_SCRIPT_PATH, "dlls/windows/Storm.dll")
+    _LINUX_STORM = os.path.join(_SCRIPT_PATH, "dlls/linux/libstorm.so.9.22.0")
```

Comment:
Are there other CPU architectures (e.g. 32 vs 64 bit) for Linux distributions?  If so, maybe rename this to `_LINUX_STORM_X86_64`.  

Or is it really unlikely to support a 32-bit distribution?

Reply:
Sure I will change the name.